backend/Squirrel.Shared/Middleware/GenericExceptionHandlerMiddleware.cs

```diff
@@ -1,8 +1,10 @@
-﻿using Newtonsoft.Json;
+﻿using Microsoft.Extensions.Logging;
+using Newtonsoft.Json;
 using Newtonsoft.Json.Serialization;
-using Squirrel.Core.Common.Extensions;
+using Microsoft.AspNetCore.Http;
+using Squirrel.Shared.Extensions;
 
-namespace Squirrel.Core.WebAPI.Middlewares;
+namespace Squirrel.Shared.Middleware;
```

Comment:
It's better to rename it to the plural noun.

Reply:
done